submission/pipelines/grab_pipeline_by_id.py

```diff
@@ -0,0 +1,22 @@
+import json
+import pymongo
+from bson import json_util
+
+real_mongo_port = 12345
+lab_hostname = "machine"
+
+
+
+mongo_client = pymongo.MongoClient(lab_hostname, real_mongo_port)
+
+pipelines_to_grab = ["0e5b9a6e-df51-4486-914a-8d862431b81d"]
+
+
+for pipeline_id in pipelines_to_grab:
```

Comment:
This should be made into a function. I am not sure how this file is used.

Reply:
It's not - I deleted it in the next PR, I can delete it in this one